rootfs/common/INSTALL_PREFIX/scripts/logpull/collect/qca.sh

```diff
@@ -95,7 +95,7 @@ collect_qcawl()
 
 collect_acceleration()
 {
-    [ -e /sys/kernel/debug/ecm/ecm_nss_ipv4 ] && collect_cmd ecm_dump.sh
+    [ -e /sys/kernel/debug/ecm/ecm_nss_ipv4 -o -e /sys/kernel/debug/ecm/ecm_ppe_ipv4 ] && collect_cmd ecm_dump.sh
```

Comment:
Is this safe against older SDKs?

Reply:
I checked on Hawkeye running spf 11.4, and it works as expected.